omnibot/services/slack/__init__.py

```diff
@@ -123,16 +123,31 @@ def update_conversations(bot, team):
 
 def update_channel(bot, channel):
     redis_client = omniredis.get_redis_client()
-    redis_client.hset(
-        f"channels:{bot.team.name}",
-        channel["id"],
-        json.dumps(channel),
-    )
-    redis_client.hset(
-        f"channelsmap:{bot.team.name}",
-        channel["name"],
-        channel["id"],
-    )
+    if channel["name"]:
```

Comment:
this could be key-error if `name` does not exist in the dict right? should it be `if channel.get("name") is not None`?

Reply:
Oh you're right, thanks for catching that!